c/include/libsbp/cpp/message_handler.h

```diff
@@ -13,11 +13,12 @@
 #ifndef SBP_CPP_MESSAGE_HANDLER_H_
 #define SBP_CPP_MESSAGE_HANDLER_H_
 
-#include <cassert>
 #include <array>
+#include <cassert>
+#include <cstddef>
```

Comment:
added this in since down below we used `size_t` which isn't guaranteed to be imported by any of the above header files (based on the documentations)